zips/zip-0312.rst

```diff
@@ -228,6 +191,14 @@ that the given SIGHASH matches the data sent from the Coordinator, or compute th
 SIGHASH themselves from that data. However, the specific mechanism for that process
 is outside the scope of this ZIP.
```

Comment:
I think the mechanism for using FROST for transaction spend auth should _not_ be outside the scope of this ZIP, given that this ZIP is "FROST for Spend Authorization Multisignatures" (rather than the smaller-scoped "Re-randomized FROST"). However, I agree that most of the content that mechanism would entail is what would go into creating and checking PCZTs, so what we should do here is incorporate by reference the ZIP that eventually contains that specification (https://github.com/zcash/zips/issues/693), and then either use it as appropriate directly in this section, or divide the specification into "Re-randomized FROST" (for more general consumption) and "Using Re-randomized FROST for Zcash transactions" (which then augments it further with the Zcash-specific logic like PCZT usage).

Reply:
I agree, let's leave this pending until we get more concrete on PCZTs